modules/conflict_resolver/jsx/conflictResolverIndex.js

```diff
@@ -0,0 +1,293 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'Loader';
+import {Tabs, TabPane} from 'Tabs';
+import FilterForm from 'jsx/FilterForm';
+import DynamicDataTable from 'jsx/DynamicDataTable';
+/**
+ * This file contains the React classes for conflict resolver
+ * module.
+ */
+
+/**
+ * This is the React class for Unresolved Conflicts
+ */
+class UnresolvedConflictsPane extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      Data: this.props.data,
+    };
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  updateFilterState(filter) {
+    this.setState({filter});
+  }
+
+  formatColumn(column, cell, rowData, rowHeaders) {
+    if (loris.hiddenHeaders.indexOf(column) > -1) {
+      return null;
+    }
+    // Create the mapping between rowHeaders and rowData in a row object.
+    let row = {};
+
+    rowHeaders.forEach(function(header, index) {
+      row[header] = rowData[index];
+    });
+
+    if (column === 'Correct Answer') {
+      const value1 = row.Value1;
+      const value2 = row.Value2;
+      const hash = row.Hash;
+      return <td>
+        <select name={hash} className='form-control input-sm'>
+          <option value='none'>Unresolved</option>
+          <option value='1'>{value1}</option>
+          <option value='2'>{value2}</option>
+        </select>
+      </td>;
+    }
+    return <td>{cell}</td>;
+  }
+  // Render the HTML
+  render() {
+    return (
+      <TabPane Title='' {...this.props}>
+        <form method='post'
```

Comment:
any chance we could use the `<FormElement />` component here insteaad?

Reply:
@zaliqarosli I think the FormElement cannot take a DynamicDataTable component? I could modify the FormElement component if you want me to but I think @HenriRabalais thought it would be good to merge this PR since the new FilterTable component needs to be worked on to include the column with select options.